improc/alignment.py

```diff
@@ -392,8 +392,10 @@ def run( self, source_image, target_image ):
                 target_wcs.provenance,
             ],  # this does not really matter since we are not going to save this to DB!
         )
-        warped_image.provenance.update_id()
-        warped_image.header['original_image_id'] = source_image.id  # add this only for aligned images for verification
+        warped_image.provenance_id = warped_image.provenance.id  # make sure this is filled even if not saved to DB
+        warped_image.header['original_image_id'] = source_image.id
```

Comment:
I liked the "additional_data" suggestion you had on the previous PR, or some other name besides "header" (to avoid confusion with raw_header).  Make this an Issue if you haven't already, because it's not worth trying to find and fix all of the instances of header in this current PR.

Reply:
Created an issue.